### PR TITLE
updating README and adding sample env file

### PR DESCRIPTION
**Pull request by horeaporutiu**
- update readme w/ template
- add sample env file
- add demo

### CODE REVIEW

File: examples/nextjs-full-stack/README.md [hunk 1]
```diff
@@ -1,64 +1,94 @@
...
+- [License](#license)
 
-1. Create a [Developer team in Miro](https://developers.miro.com/docs/create-a-developer-team).
-2. Create an [app in Miro](https://developers.miro.com/docs/build-your-first-hello-world-app#step-1-bootstrap-the-hello-world-app).
-3. Create a board in Miro that you'd like to manipulate with the REST API.
+# ⚙️ Included Features <a name="features"></a>
 
-## How to start
+- [Miro Node.js Client Library](https://developers.miro.com/docs/web-sdk-reference)
+  - [miro.isAuthorized](https://miroapp.github.io/api-clients/classes/index.Miro.html#isAuthorized)
```
Comment (imxena) [inline examples/nextjs-full-stack/README.md:24]: Are we exposing the github pages url ?

Comment (imxena) [reply examples/nextjs-full-stack/README.md:24]: how come it's not part of the dev portal ?

Comment (imxena) [reply examples/nextjs-full-stack/README.md:24]: I will ping you separately. This in unrelated to the PR. :D

**Review by imxena — APPROVED** [review]

File: examples/nextjs-full-stack/README.md (hunk 1 above)
Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:24]: This is just where the Node.js client is hosted right @imxena ?

I thought it shouldn't be an issue, but please let me know if I am wrong here. I just wanted to point them in the direction of the docs for the API methods which are used for this example.

Comment (imxena) [reply examples/nextjs-full-stack/README.md:24]: @horeaporutiu  I was surprised cz I also saw the github page url exposed directly for the first time & thought that it should have been a developer portal (developers.miro.com) url. It is indeed the url to share for api client docs for now.

**Pull request merged**